app/graphql/types/customers/object.rb

```diff
@@ -124,6 +126,17 @@ def billing_configuration
           document_locale: object&.document_locale
         }
       end
+
+      def shipping_address
```

Comment:
Maybe this part could be extracted into a method on the customer model as it seems it will be also used on the serializer side

Reply:
Moved on the model